scripts/process_crashes.sh

```diff
@@ -30,8 +30,14 @@ function cleanup {
 trap cleanup EXIT
 
 if [[ $# -eq 0 ]]; then
-    echo "Usage: process_crashes.sh CRASHID [CRASHID ...]"
-    exit 1
+    if [ -t 0 ]; then
+        # If stdin is a terminal, then there's no input
+        echo "Usage: process_crashes.sh CRASHID [CRASHID ...]"
+        exit 1
+    fi
+
+    # stdin is not a terminal, so pull the args from there
+    set -- ${@:-$(</dev/stdin)}
```

Comment:
So what this does is set the positional args to `$@` (the positional args) if there are any and default to data from stdin if not `$(</dev/stdin)`.